fe-nc-news-react/src/api.js

```diff
@@ -22,12 +22,16 @@ function patchArticleVote(newVote, id) {
 }
 
 function postNewComment(id, newComment) {
-    console.log(id, newComment);
     return axios.post(
       `https://j-nc-news.onrender.com/api/articles/${id}/comments`, newComment
     ).catch()
 }
+
+function deleteComment(id) {
+    return axios.delete(`https://j-nc-news.onrender.com/api/comments/${id}`)
+    .catch()
```

Comment:
we dont want to have a catch in the middle of our promise chain as this will prevent us from being able to deal with errors inside our component